src/main/java/com/google/api/codegen/viewmodel/SimpleInitCodeLineView.java

```diff
@@ -49,6 +52,8 @@ public static Builder newBuilder() {
 
     public abstract Builder doc(List<String> val);
 
+    public abstract Builder isFirst(boolean val);
```

Comment:
It would be better if this variable were more descriptive of its intention. Maybe, `leadingNewline`, which has the added benefit of being usable in other cases where a leading newline might be needed?